contents/blog/the-posthog-array-1-40-0.md

```diff
@@ -0,0 +1,71 @@
+---
+date: 2022-08-29
+title: 'Array 1.40.0: What's new in PostHog?'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? [Subscribe to HogMail, our newsletter](/newsletter), which we send out every two weeks!
+
+> Running a self-hosted instance? Check out our [guide to upgrading PostHog](/docs/runbook/upgrading-posthog).
+
+## PostHog 1.40.0 release notes
+
+**Release highlights:**
+
+- [Highlight One](#highlight-one)
+- [Highlight Two](#highlight-two)
+- [Highlight Three](#highlight-three)
+
+### Highlight One
+
+Some copy goes here. 
+
+### Highlight Two
+
+Some copy goes here. 
+
+### Highlight Three
+
+Some copy goes here. 
+
+### Other improvements & fixes
+Version 1.40 also adds hundreds of other improvements and fixes, including...
+
+- **Improvement:** Minor feature one
```

Comment:
@neilkakkar @EDsCODE As above, any minor updates can be added here. Appreciate these usually come in as last-minute adds.